Taskfile.yml

```diff
@@ -23,18 +23,20 @@ vars:
     prql-lang.prql-vscode
     rust-lang.rust-analyzer
   cargo_crates: |
-    bacon
-    cargo-audit
     cargo-insta
-    cargo-release
-    default-target
     mdbook
     mdbook-admonish
     mdbook-footnote
     mdbook-toc
     wasm-bindgen-cli
     wasm-opt
     wasm-pack
+  # Crates for administering the repo
+  cargo_crates_admin: |
+    bacon
+    cargo-audit
+    cargo-release
```

Comment:
There is a question about the selection of tools. Why did you choose these?

Reply:
The others are required for building & testing the repo — or are there any that I misallocated? 

I can add a comment with the principle